.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -88,6 +88,14 @@ jobs:
         conda config --set solver libmamba
     displayName: "Add Conda to PATH"
 
+  - bash: |
+      grep MOLSYM psi4/src/read_options.cc
+      if [[ "$(Agent.JobName)" == "Linux Builds gcc_latest" ]]; then
+        sed -i 's;\(options\.add_str("SDSALC_PACKAGE", "\)INTERNAL\(", "INTERNAL MOLSYM"\);\1MOLSYM\2;' psi4/src/read_options.cc
+      fi
+      grep MOLSYM psi4/src/read_options.cc
+    displayName: "Special Operation: test MolSym"
```

Comment:
This isn't all hooked up at present. This and a line in read_options need to be made to match. The intention is to have an Azure lane that runs molsym for every findif by forcing the default. Usually for addons we using `using('addon')` to add new tests or parametrize old ones. This alternate way is for when there's too many tests (all those with the `findif` marker).